<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="intput_string">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_intSring">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>源字符串：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_outString">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>转换结果：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_string">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>541</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>可用转换器：</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>151</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="add_transformer">
           <property name="text">
            <string>增加&gt;&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_transformer">
           <property name="text">
            <string>去除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_all_transformer">
           <property name="text">
            <string>全部去除</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>0</y>
          <width>141</width>
          <height>131</height>
         </rect>
        </property>
        <property name="title">
         <string>变换器链：</string>
        </property>
        <widget class="QTextEdit" name="transformer_list_text">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>141</width>
           <height>111</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QCheckBox" name="checkBoxSmallToBig">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>转换为大写</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxBigToSmall">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>转换为小写</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxDeleleSpace">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>剔除掉空格</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="application">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
